logging/cluster-logging-release-notes.adoc

```diff
@@ -8,34 +8,65 @@ toc::[]
 
 // Release Notes by version
+
+[id="cluster-logging-release-notes-5-2-0"]
+== OpenShift Logging 5.2.0
+
+This release includes link:https://access.redhat.com/errata/RHSA-TBD[RHSA-TBD OpenShift Logging Bug Fix Release 5.2.0].
+
+[id="openshift-logging-5-2-0-new-features-and-enhancements"]
+=== New features and enhancements
+
+* TBD-description
+(link:https://issues.redhat.com/browse/LOG-TBD[LOG-TBD])
+
+* With this update, if you use the Fluentd forward protocol to forward log data over a TLS-encrypted connection, you can now use a password-encrypted private key file and specify the passphrase in the Cluster Log Forwarder configuration. For more information, see xref:../logging/cluster-logging-external.html#cluster-logging-collector-log-forward-fluentd_cluster-logging-external[Forwarding logs using the Fluentd forward protocol]
+(link:https://issues.redhat.com/browse/LOG-1525[LOG-1525])
+
+* With this update, you can use a username and password to authenticate a log forwarding connection to an external Elasticsearch instance. For example, if you cannot use mutual TLS (mTLS) because a third-party operates the Elasticsearch instance, you can use HTTP or HTTPS and set a secret that contains the username and password. For more information see xref:../logging/cluster-logging-external.adoc#cluster-logging-collector-log-forward-es_cluster-logging-external[Forwarding logs to an external Elasticsearch instance].
+(link:https://issues.redhat.com/browse/LOG-974[LOG-1022])
+
+
+
+[id="openshift-logging-5-2-0-deprecated-removed-features"]
+=== Deprecated and removed features
+
+Some features available in previous releases have been deprecated or removed.
+
+Deprecated functionality is still included in OpenShift Logging and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments.
+
+[id="openshift-logging-5-2-0-tbd"]
+==== TBD-title
+
+TBD-description
+
+[id="openshift-logging-5-2-0-bug-fixes"]
+=== Bug fixes
+
+* TBD-description
+(link:https://issues.redhat.com/browse/LOG-TBD[LOG-TBD])
+
+* Before this update, if you redeployed a full Elasticsearch cluster, it got stuck in an unhealthy state, with one non-data node running and all other data nodes shut down. This happened because new certificates prevented the Elasticsearch Operator from scaling down the non-data nodes of the Elasticsearch cluster. With this update, Elasticsearch Operator can scale all the data and non-data nodes down and then back up again, so they load the new certificates. The Elasticsearch Operator can reach the new nodes after they load the new certificates.
+(link:https://issues.redhat.com/browse/LOG-1524[LOG-1524])
```

Comment:
```suggestion
* In previous releases, if you redeployed a full Elasticsearch cluster, it might become stuck in an unhealthy state, with one non-data node running and all other data nodes shut down. This was caused by new certificates preventing the Elasticsearch Operator from scaling down the non-data nodes of the Elasticsearch cluster. With this update, The Elasticsearch Operator can scale all the data and non-data nodes down and then back up again, so that the nodes can load the new certificates. The Elasticsearch Operator can reach the new nodes after they load the new certificates.
(link:https://issues.redhat.com/browse/LOG-1524[LOG-1524])
```

Reply:
Some suggestions for readability - this would also be a good place to add an attribute for the Operator name maybe?

Reply:
Good suggestion. I'll add an attribute for this and other operators in a later release: https://issues.redhat.com/browse/RHDEVDOCS-3238

Reply:
Updated. Thanks.